common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -119,6 +119,13 @@ describe 'Problem', ->
         @problem.render(problem_content_default)
         expect(@problem.$('.problem-progress').html()).toEqual "(2/2 points)"
 
+    describe 'with valid status and and string containing an integer like "0" for detail', ->
```

Comment:
I corrected this.

I added a new test that forces the failure state, I will add the comment and squash some commits